index.html

```diff
@@ -259,6 +259,68 @@ <h2>LDP Containers (LDPC)</h2>
             predicate, and thus triples with this predicate do not necessarily imply containment.
           </blockquote>
         </section>
+        <section id="ldpdc">
+          <h2>LDP Direct Containers</h2>
+          <p>
+            Implementations that support the creation and management of LDP Direct Containers ([[!LDP]]
+            <a href="https://www.w3.org/TR/ldp/#ldpdc">5.4</a>)
+            use the <code>ldp:membershipResource</code> property to express the membership constant URI (per
+            [[!LDP]] <a href="https://www.w3.org/TR/ldp/ldpdc-containres">5.4.1.3</a>), and use either the
```

Comment:
Link does not resolve.  Should be https://www.w3.org/TR/ldp/#ldpc ?

Reply:
Oh, yes, missing #... should be https://www.w3.org/TR/ldp/#ldpdc-containres . Thanks, will fix